.github/workflows/proxy-warming.yml

```diff
@@ -0,0 +1,17 @@
+name: Renew documentation
+
+on:
+  release:
+    types:
+      - created
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - '**/v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
nice, but this will not do it for the future alpha or beta releases 🤔 

or any releases with letters in them